lib/generators/maestrano/templates/maestrano.rb

```diff
@@ -121,8 +121,8 @@
   # you run 'rake maestrano:install' and is available under
   # <rails_root>/app/controllers/maestrano/account/
   #
-  # config.webhook.account.groups_path = '/maestrano/account/groups/:id',
-  # config.webhook.account.group_users_path = '/maestrano/account/groups/:group_id/users/:id',
+  config.webhook.account.groups_path = '/maestrano/account/groups/:id/default'
+  config.webhook.account.group_users_path = '/maestrano/account/groups/:group_id/users/:id/default'
```

Comment:
Those config changes are not compatible with the maestrano-ruby default config..